### PR TITLE
install packages with yarn instead of npm, when is is available :)

### PR DESCRIPTION
**Pull request by chrishelgert**
enhancement for our script/install.sh :)
now it will check if the user have yarn available, when yes it will use yarn for faster download :smirk: else it will use the standard npm-install

### CODE REVIEW

File: scripts/install.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
+const caniuseYarn = require('@danielbayerlein/caniuse-yarn');
 const executeAll = require('./helper/executeAll');
 const projects = require('./helper/projects');
 
-executeAll('npm', ['install'], projects);
+if (caniuseYarn()) {
+  executeAll('yarn', ['install'], projects);
```
Comment (chrishelgert) [inline scripts/install.js]: i know we could also use a ternary, but it think in this case a if - else is more readable ;)

File: package.json [hunk 2]
```diff
@@ -18,5 +18,8 @@
     "ghooks": {
       "pre-commit": "npm test"
     }
+  },
+  "dependencies": {
+    "@danielbayerlein/caniuse-yarn": "^1.0.0"
```
Comment (danielbayerlein) [inline package.json]: I think it's a dev-dependency

File: scripts/install.js (hunk 1 above)
Comment (danielbayerlein) [inline scripts/install.js]: Looks amazing, or? 👯 

```javascript
executeAll(caniuseYarn() ? 'yarn' : 'npm', ['install'], projects);
```

**Comment by chrishelgert** [comment]
@danielbayerlein as you wish :stuck_out_tongue:

**Pull request merged**